.github/workflows/pull_request_verification.yml

```diff
@@ -5,10 +5,59 @@ on:
     branches: '**'
 
 jobs:
-  build:
 
+  branch_name:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+          cache: gradle
+
+      - name: Check correct branch name
+        run: |
+          tag=${GITHUB_HEAD_REF#refs/*/v}
+          if [[ $tag =~ [a-z]{2,}\/[0-9]+-[A-Za-z-]+$ ]]
+          then
+            echo "::notice::Tag is valid — $tag"
+            echo "{tag}={GITHUB_HEAD_REF#refs/*/v}" >> $GITHUB_OUTPUT
```

Comment:
i am wondering if we really need message that the branch name is valid, maybe we should only receive message when the branch name is invalid.

Reply:
I deleted this message.